test/property-access.civet

```diff
@@ -279,31 +279,39 @@ describe "property access", ->
       ---
       x.-1
       ---
-      x.at(-1)
+      x[x.length-1]
     """
 
     testCase """
       optional negative
       ---
       x?.-1
       ---
-      x?.at(-1)
+      x?.[x.length-1]
     """
 
     testCase """
       possessive negative
       ---
       x's -1
       ---
-      x.at(-1)
+      x[x.length-1]
     """
 
     testCase """
       optional negative
       ---
       x?.-1
       ---
-      x?.at(-1)
+      x?.[x.length-1]
```

Comment:
This test seems to be a duplicate of above. (Probably my bad.) Maybe it's supposed to be testing `x?'s -1`?